tests/tests/parsers/code_22e9.log

```diff
@@ -0,0 +1,3 @@
+2024-10-15T12:32:31.521795 000 RQ --- 18:130140 32:022222 --:------ 22E9 001 00        # {}
+2024-10-15T12:32:31.538027 082 RP --- 32:022222 18:130140 --:------ 22E9 004 00010000  # {'percent_2': 0.005, 'percent_4': 0.0, 'percent_6': 0.0}      # Vasco D60
+2024-10-15T22:59:02.967986 084 RP --- 32:022222 18:130140 --:------ 22E9 004 00010701  # {'percent_2': 0.005, 'percent_4': 0.035, 'percent_6': 0.005}  # idem
```

Comment:
This is a very small HRU unit, with just a few parts. Most of the time it reports one of these 2 states. When the mains power is switched on it is in pairing/startup state for about 10 mins. So that could be what this is, 4 settings eg. the base speed/level that later are inaccessible/off. I will try to change it tomorrow